tools/clang/lib/Sema/SemaOverload.cpp

```diff
@@ -5109,7 +5109,14 @@ static bool CheckConvertedConstantConversions(Sema &S,
   case ICK_TransparentUnionConversion:
   case ICK_Writeback_Conversion:
   case ICK_Zero_Event_Conversion:
-  case ICK_Flat_Conversion: // HLSL Change
+ // HLSL Change Start
+  case ICK_Flat_Conversion:
```

Comment:
change in behavior: These cases used to hit `llvm_unreachable`, but now hit `false`. Is this the correct behavior?